spec/typescript/exceljs.spec.ts

```diff
@@ -1,7 +1,10 @@
 import 'regenerator-runtime/runtime';
 
 import { expect } from 'chai';
-import ExcelJS from '../../index';
+import _ExcelJS from '../../index';
+import type * as ExcelJSModule from '../../exceljs';
+
+const ExcelJS = _ExcelJS as unknown as typeof ExcelJSModule;

```

Comment:
Type checking wasn't currently being picked up by ts-node due to `../../index` resolving to the untyped index files rather than `../../index.d.ts`.

https://stackoverflow.com/questions/62490423/consume-index-d-ts-declaration-file-in-source-code

> It looks like you have a collision in your module resolution.
>
> If you have a file called index.ts, then index.d.ts won't be automatically imported.
> 
> TypeScript will automatically import .d.ts files, UNLESS their name collides with other files, where the rules for what files will collide are somewhat described here: https://www.typescriptlang.org/docs/handbook/module-resolution.html .

For that reason, I've renamed `index.d.ts` to `exceljs.d.ts` (and updated the corresponding package.json fields etc).

It still needs to be imported in this weird way for the test though. Possibly there's a better fix, but I couldn't figure one out.